common/templates/table.html

```diff
@@ -3,18 +3,18 @@
         <!-- sortable column header -->
         <thead>
             <tr>
-                <th nowrap ng-repeat="col in tableModel.columns">
-                    <span class="table-column-displayname" ng-click="sortBy(col.name)">{{col.displayname}}</span>
-                    <span ng-show="tableModel.sortedBy==col.name && tableModel.sortOrder=='asc'" ng-click="toggleSortOrder()" class="fa fa-sort-amount-asc" ng-style="{'cursor': 'pointer'}"></span>
-                    <span ng-show="tableModel.sortedBy==col.name && tableModel.sortOrder=='desc'" ng-click="toggleSortOrder()" class="fa fa-sort-amount-desc" ng-style="{'cursor': 'pointer'}"></span>
-                    <span ng-show="tableModel.sortedBy!==col.name" class="fa fa-fw fa-sort" ng-click="sortby(col.name)" ng-style="{'cursor': 'pointer'}"></span>
+                <th nowrap ng-repeat="col in vm.columns track by $index">
+                    <span class="table-column-displayname" ng-click="sortby({colname: col.name})">{{col.displayname}}</span>
+                    <span ng-show="vm.sortby==col.name && vm.sortOrder=='asc'" ng-click="toggleSortOrder()" class="fa fa-sort-amount-asc" ng-style="{'cursor': 'pointer'}"></span>
+                    <span ng-show="vm.sortby==col.name && vm.sortOrder=='desc'" ng-click="toggleSortOrder()" class="fa fa-sort-amount-desc" ng-style="{'cursor': 'pointer'}"></span>
+                    <span ng-show="vm.sortby!==col.name" class="fa fa-fw fa-sort" ng-click="sortby({colname: col.name})" ng-style="{'cursor': 'pointer'}"></span>
                 </th>
             </tr>
         </thead>
 
         <!-- rows -->
-        <tr class="table-row" ng-repeat="row in tableModel.rowset track by $index">
-            <td ng-repeat="col in tableModel.columns">{{row.values[$index]}}</td>
+        <tr ng-repeat="row in vm.rowValues track by $index" ng-click="gotoRowLink({index : $index})" ng-style="{'cursor': 'pointer'}">
+            <td ng-repeat="val in row track by $index">{{val}}</td>
```

Comment:
In removing the class attribute from the <tr>, the tests can no longer find this element. That's why they are failing.